nuxeo-core/nuxeo-core-event/src/test/java/org/nuxeo/ecm/core/work/TestInMemoryWorkManager.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * (C) Copyright 2012-2018 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Florent Guillaume
```

Comment:
hmm

Reply:
I wanted him to get the credit of my work... if you don't think he deserves it...